src/modules/core/components/BannedTag/BannedTag.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { HTMLAttributes } from 'react';
+import { MessageDescriptor } from 'react-intl';
+
+import Icon from '~core/Icon';
+import Tag from '~core/Tag';
+
+import styles from './BannedTag.css';
+
+const displayName = 'BannedTag';
+
+interface Props extends HTMLAttributes<HTMLSpanElement> {
+  text?: MessageDescriptor | string;
+}
+
+const BannedTag = ({ text = 'Banned' }: Props) => {
```

Comment:
this shouldn't be just a string `Banned`, it should be created via react-intl. Please, check in other components how we do it and apply it here

Reply:
I've already added a global id for this.

So you can just use `text={{ id: 'label.banned' }}`

Reply:
Done, 
`                      <BannedTag text={{ id: 'label.banned' }} />
`